addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -265,8 +265,8 @@
                 <field name="holiday_type" invisible="1" readonly="state not in ['confirm', 'draft']"/>
                 <field name="leave_type_request_unit" invisible="1"/>
                 <div class="o_hr_leave_content row my-n4">
-                    <div class="o_hr_leave_column col_left col-md-6 col-12">
-                        <div name="title" class="o_hr_leave_title" invisible="1">
+                    <div class="o_hr_leave_column col_left col-md-6 col-12 py-3">
```

Comment:
commit 771dc529383f66446aa307baada638a0c9ce192c
Is this padding really necessary? Relying on the form_sheet padding seems sufficient

Reply:
See https://github.com/odoo/odoo/pull/142203#discussion_r1474461579